WORKSPACE

```diff
@@ -169,32 +169,6 @@ dev_env_tool(
     win_tool = "msys2-20180531",
 )
 
-# c2hs
-nixpkgs_package(
-    name = "c2hs",
-    attribute_path = "ghcWithC2hs",
```

Comment:
Can we also kill the `ghcWithC2hs` Nix expression and its dependencies in `nix/`?

Reply:
Created a ticket: https://github.com/digital-asset/daml/issues/528

Reply:
Thanks!